### ISSUE TITLE
Tag releases

### ISSUE THREAD

**Issue by tordans** [beerose/simple-zustand-devtools#26]
Is there a way to see which code was shipped in a specific release? I cannot find any [tags](https://github.com/beerose/simple-zustand-devtools/tags), [branches](https://github.com/beerose/simple-zustand-devtools/branches/all) nor [releases](https://github.com/beerose/simple-zustand-devtools/releases).

Right now, it looks like the only way to compare releases is to look at the [npm release history](https://www.npmjs.com/package/simple-zustand-devtools?activeTab=versions) and compare commits. Am I missing something? 

It would be great to have releases as tags in Github.

**Comment by beerose** [comment]
Yeah, that's a good point. I'll start doing it from now on. For now, I tagged the last three releases.
